const FaultyComponent = () => {
  throw new Error('This is a test error!');
};

export default FaultyComponent;
